sections/related_works.tex

```diff
@@ -1 +1,27 @@
 \section{Related Work}\label{sec:related-works}
+In this section, we investigate existing literature concerning \gls{lda} and similar topic models, as well as relevant work related to random walk.
+
+\citet{lda} describes a generative statistical model, which is heavily used within topic modelling. 
+The intuition behind the \gls{lda} topic model is that a document has an underlying distribution of topics of which it consists. 
+This intuition is set up by ... \todo[inline]{Ved ikke umiddelbart hvor sætningen vil hen.}
```

Comment:
```suggestion
This model is used widely within the field of topic modeling and is somewhat considered a standard.
```

Reply:
\todo{ref to lda section for more detail.}